tasks/main.yml

```diff
@@ -23,6 +23,7 @@
     uid: "{{item.uid}}"
     home: "{{ item.home | default('/home/' + item.username) }}"
     createhome: "{{'yes' if users_create_homedirs else 'no'}}"
+    update_password: "{{ 'always' if (item.force_passwd_update is defined and item.force_passwd_update) else 'on_create' }}"
```

Comment:
The role should fall back to the user module's default of `always` (as it currently does) when `update_password` is undefined.

    update_password: "{{item.update_password | default(omit)}}"